cdc/kv/metrics.go

```diff
@@ -67,6 +67,13 @@ var (
 			Name:      "channel_size",
 			Help:      "size of each channel in kv client",
 		}, []string{"id", "channel"})
+	clientRegionTokenSize = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "ticdc",
```

Comment:
maybe extract the `ticdc`  `kvclient` in
```
Namespace: "ticdc",
Subsystem: "kvclient",
```
 to const ?

Reply:
Yep they are constant values, but I will not change them to constants in this PR, since too many literal constants are used in metric names, maybe in another PR we can address it